Makefile

```
@@ -8,7 +10,7 @@ run: ## Run the game
 	@swipl --quiet -t "main($(level), \"$(file)\", $(players), $(fac))" -s game.pl 
     
 test: ## Run tests
-	@swipl --quiet -t "load_test_files([]), run_tests." -s game.pl 
+	@swipl -t "load_test_files([]), run_tests." -s game.pl 
```

Change to top level by goals in order to hide the swipl welcome message
```suggestion
	@swipl -g "load_test_files([]), run_tests." -g halt -s game.pl 
```

---

Done in last commit